wiki_articles/pi.md

```diff
@@ -0,0 +1,27 @@
+<!-- alias pi -->
+# How Do I Get Pi and Other Math Constants in C/C++?
+
+## C++
+```cpp
+// C++20:
+constexpr double pi = std::numbers::pi; // from <numbers>
+constexpr T      pi = std::numbers::pi_v<T>;
+// C++17 or lower:
+const double pi = std::acos(-1); // from <cmath>
+const T      pi = std::acos(T(-1));
+// note: all variables can be constexpr since C++26
```

Comment:
It'd be better to leave out the c++26 stuff from a beginner article for now. There's no "since c++26" for another few years :)

Reply:
I've thought about it, but I think it's worth keeping purely out of peeking academic interest. It's just a note at the end of the code block, and so it doesn't cost too much.

It also implies that since C23/C++26, it's possible to write
```cpp
constexpr double pi = acos(-1);
constexpr T pi = acos((T) -1);
```
... in both languages, and have perfect compatibility.

Part of the goal of the article is to inform users of more modern features, and that includes advanced users. Therefore it's in the spirit of the article to reach ahead a little bit.

Reply:
Sounds good. Let's update it to read "after c++26" / "after c23" though.